models/card.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const cardSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 30,
+  },
+  link: {
+    type: String,
+    required: true,
+    validate: {
+      validator: (item) => {
+        validator.isUrl(item);
```

Comment:
Нужно исправить: при создании карточки возвращается 500.
```suggestion
        validator.isURL(item);
```